full/src/test/java/apoc/load/LoadDirectoryTest.java

```diff
@@ -0,0 +1,108 @@
+package apoc.load;
+
+import apoc.util.TestUtil;
+import org.junit.*;
+import org.junit.rules.TemporaryFolder;
+import org.neo4j.configuration.GraphDatabaseSettings;
+import org.neo4j.dbms.api.DatabaseManagementService;
+import org.neo4j.graphdb.GraphDatabaseService;
+import org.neo4j.internal.helpers.collection.Iterators;
+import org.neo4j.test.TestDatabaseManagementServiceBuilder;
+
+import java.io.File;
+import java.util.List;
+import java.util.Map;
+
+import static apoc.ApocConfig.*;
+import static apoc.util.TestUtil.testResult;
+import static org.junit.Assert.*;
+
+
+public class LoadDirectoryTest {
+
+    @ClassRule
+    public static TemporaryFolder temporaryFolder = new TemporaryFolder();
+
+    private static GraphDatabaseService db;
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+        File importFolder = new File(temporaryFolder.getRoot() + "/import Dir");
+        DatabaseManagementService databaseManagementService = new TestDatabaseManagementServiceBuilder(importFolder).build();
+        db = databaseManagementService.database(GraphDatabaseSettings.DEFAULT_DATABASE_NAME);
+
+        TestUtil.registerProcedure(db, LoadDirectory.class);
+
+        // create temp files and subfolder
+        temporaryFolder.newFile("Foo.csv");
+        temporaryFolder.newFile("Bar.csv");
+        temporaryFolder.newFile("Baz.xls");
+        temporaryFolder.newFolder("import Dir/subfolder");
```

Comment:
```suggestion
        temporaryFolder.newFolder(importDir + File.separator + "TestCsv1.csv");
```
la stessa cosa anche per gli altri